queries.sql

```diff
@@ -7,3 +7,27 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+START TRANSACTION;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+
```

Comment:
- [ ] @shaaibu7 kindly add a query to show changes made to the species column, also commit the transaction and then add a query to verify that the change persists after the commit.